tests/unit/utils.py

```diff
@@ -70,6 +70,7 @@ def project_from_dict(project, profile, packages=None, selectors=None, cli_vars=
         project_root=project_root,
         project_dict=project,
         packages_dict=packages,
+        dependent_projects_dict={},
```

Comment:
We need to make this same change in the adapter repos, since they copy-paste `tests/unit/utils.py` for defining their own unit tests
- https://github.com/dbt-labs/dbt-snowflake/pull/661/commits/ad27811dd36506c01467dd441ed123570ad79c1f
- https://github.com/dbt-labs/dbt-bigquery/pull/774/commits/45c57574a0464a53ff84004509fd81deae7615b6
- (same for redshift + spark)